course_discovery/apps/course_metadata/tests/test_signals.py

```diff
@@ -579,7 +579,7 @@ def setUpTestData(cls):
         )
 
     def test_draft_does_not_collide_with_draft(self):
-        with self.assertNumQueries(77, threshold=0):
+        with self.assertNumQueries(71, threshold=1):
```

Comment:
With `threshold=1`, the assertion is now fuzzy with a +/- 1. Is the test actually flaky/fuzzy? Or do we need to just adjust 71 to 70 or 72 instead? I generally prefer to see `threshold=0` if we can make that work.

But if this test did become flakier because of some new elasticsearch oddity, fine.

Reply:
I don't like it either and locally everything works without threshold. On the Travis is not so, often problems arise in different tests and one or more extra requests appear. I would also really like `threshold=0`. 
I can say that it has nothing in common with some new elasticsearch oddity.

By the way, look at this commit https://github.com/raccoongang/course-discovery/commit/c101498d24eba285717670b4bfdbaa7f2e310b2d.
I see that you also had problems with this 🤔

I did a little analysis of why this happens in CI and I see that practically always an extra request if it happens then this
```
SELECT `waffle_switch`.`id`, `waffle_switch`.`name`, `waffle_switch`.`active`, `waffle_switch`.`note`, `waffle_switch`.`created`, `waffle_switch`.`modified` FROM `waffle_switch` WHERE `waffle_switch`.`name` = 'publish_course_runs_to_marketing_site'
```
Why does `waffle_switch` show up on CI and not executed when running it locally? I don't know... This is a mystery so far.
This is another problem that was before our work. I suggest keeping this in mind for now and leaving it that way.
What do you think?